examples/_next/src/pages/_app.tsx

```diff
@@ -15,6 +15,7 @@ import { publicProvider } from 'wagmi/providers/public';
 const livepeerClient = createReactClient({
   provider: studioProvider({
     apiKey: process.env.NEXT_PUBLIC_STUDIO_API_KEY ?? '',
+    baseUrl: 'https://livepeer.monster/api',
```

Comment:
@0xcadams Is this on a public documentation/example? If so, we should avoid directing users to use our staging environment which is internal only (especially cause we don't have any restrictions today like only allowing livepeer.org emails). Is it possible to use prod here?

Reply:
Those apps preceded by underscore are internal apps for developers, so no, it's not docs